src/witan/models/dem/ccm/components_functions.clj

```diff
@@ -69,10 +69,18 @@
                                 [:regres-coef :intercept]
                                 (fn [y i] (let [t (+ i (* y (inc end-yr-avg)))]
                                             (if (neg? t)
-                                              0
+                                              0.0
                                               t))))
         (ds/select-columns [:gss-code :sex :age trend-out]))))
 
+(defn add-years-to-fixed-methods
+  "Plots fixed rates across all projected years"
+  [fixed-projection end-yr-avg-mort last-proj-yr]
+  (prn (inc end-yr-avg-mort) (inc last-proj-yr))
```

Comment:
sorry @seb231 just realised this is wrong - this needs to be the first projection year NOT (inc end-yr-avg-mort)